magda-web-client/src/Components/Editing/ToggleEditor.tsx

```diff
@@ -56,7 +56,7 @@ export class ToggleEditor<V> extends React.Component<ToggleEditorProps<V>> {
             value = this.props.value;
         }
 
-        enabled = enabled === undefined || enabled;
+        enabled = enabled ? true : false;
```

Comment:
Tbh I think there's a few fundamental problems with how this was done originally.

1. That "enabled" isn't a great prop name (the field is "enabled" no matter what, what enabled actually does is toggle whether it can be edited.
2. That it allows for the prop to not be set.

How do you feel about renaming `enabled` to `editable` and making it mandatory (i.e. it always has to be set to `true` or `false`) so that this mistake is harder to make?

Reply:
Yeah~ I probably can rename it and make it throw an exception if its value is not a boolean~